pkg/kubelet/cm/devicemanager/topology_hints_test.go

```diff
@@ -440,7 +440,6 @@ func TestTopologyAlignedAllocation(t *testing.T) {
 				opts: &pluginapi.DevicePluginOptions{GetPreferredAllocationAvailable: true},
 			}
 		}
-
```

Comment:
I'm not opposed to this change, but it seems superfluous and not related to anything else we are doing in this PR

Reply:
Agreed. I don't know why this was changed, will revert.